apps/dotcms-ui/src/app/view/components/dot-toolbar/dot-toolbar.component.html

```diff
@@ -11,8 +11,16 @@
         <div
             class="toolbar__logo-wrapper"
             [class.toolbar__logo-wrapper--collapsed]="dotNavigationService.collapsed$ | async"
+            [class.toolbar__logo-wrapper--whitelabel]="logo"
         >
-            <span class="toolbar__logo"></span>
+            <div *ngIf="logo; else toolbarLogo">
+                <span
+                    class="toolbar__logo--whitelabel"
+                    [style.background-image]="'url(' + photo + ')'"
+                    [ngStyle]="{ 'background-image': logo, 'background-size': 'contain' }"
```

Comment:
css class?

Reply:
I think is dupe the `background-image` property here.